freqtrade/freqai/freqai_interface.py

```diff
@@ -83,6 +83,7 @@ def __init__(self, config: Config) -> None:
         self.CONV_WIDTH = self.freqai_info.get('conv_width', 1)
         if self.ft_params.get("inlier_metric_window", 0):
             self.CONV_WIDTH = self.ft_params.get("inlier_metric_window", 0) * 2
+        self.class_names: List[str] = []  # used in classification subclasses
```

Comment:
This comment suggests that this is used throughout the classifier models - but i don't think that's true?

is it intentional that we have these only implemented in the new Torch models?

Reply:
Yes it is intentional. 
We use for:

1. create Dict[str, int] maaping, to encode decode the str class names to int, as it is necessary by torch modules.
2. initialize the model with proper number of output classes

XGBoost, CatBoost have their own built-in methods for dealing with class targets of type str. This attribute could be used with any model that doesn't have built-in method.

Reply:
We can change the comment to somthing like:
# used in pytorch classification subclasses

On the other end, 
1. i think it's a good practice to declare class names in the strategy.
2. There will be a future PR with tensorboard logger, that will be excessible to all models. 
Then it might be more stable/convenient to use this attribute for classifier XGBoost/CatBoost classifiers.

Reply:
The fact that "it's intentional" is enough for me already :+1: 

I assumed that the other models have this builtin - just making sure ;)